bin/strip_whitespace

```diff
@@ -11,8 +11,7 @@ def strip_file(filename, write, report):
     # to re-add \n to a few right-stripped lines. The hit flag will keep us
     # from unecessarily re-writing files with no changes.
 
-    # without "b" all lines appear as \n terminated
-    with open(filename, 'rb') as f:
+    with open(filename, 'r') as f:
```

Comment:
This should be `open(filename, newlines='')`, else it will change newlines between windows and unix.